android/beagle/src/main/java/br/com/zup/beagle/android/setup/BeagleSdk.kt

```diff
@@ -51,4 +51,8 @@ interface BeagleSdk {
         BeagleEnvironment.application = application
         SoLoader.init(application, false)
     }
+
+    fun setInTestMode() {
```

Comment:
What do you think about add the annotation `VisibleForTesting`?
```suggestion
  @VisibleForTesting
    fun setInTestMode() {
```